overlord/ifacestate/apparmorprompting/prompting_test.go

```diff
@@ -264,8 +281,20 @@ func (s *apparmorpromptingSuite) prepManagerWithSinglePrompt(c *C, reqChan chan
 		Class:      notify.AA_CLASS_FILE,
 		Permission: notify.AA_MAY_READ,
 	}
+	// push a request
 	reqChan <- req
 
+	// which should generate a notice
+	s.st.Lock()
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
```

Comment:
timeout can be shorter I think, more like 10s on the high end. Usually sub-1s is actually fine